pipeline/cfn/testdrive.json

```diff
@@ -0,0 +1,182 @@
+            }
+        },
+
+        "InternetGateway": {
+            "Type": "AWS::EC2::InternetGateway",
+            "Properties": {
+                "Tags": [{ "Key": "Name", "Value": { "Ref": "AWS::StackName" } },
+                         { "Key": "Application", "Value": { "Ref": "AWS::StackId" } }]
+            }
+        },
+
+        "AttachGateway": {
+            "Type": "AWS::EC2::VPCGatewayAttachment",
+            "Properties": {
+                "VpcId": { "Ref": "BootstrapVPC" },
+                "InternetGatewayId": { "Ref": "InternetGateway" }
+            }
+        },
+
+        "BootstrapInstance": {
+            "Type": "AWS::EC2::Instance",
+            "Properties": {
+                "ImageId": { "Fn::FindInMap": [ "RegionConfig", { "Ref": "AWS::Region" }, "ami" ]},
+                "InstanceType": "t2.micro",
+                "IamInstanceProfile": { "Ref": "BootstrapInstanceProfile" },
+                "KeyName": {"Ref" : "AWS::NoValue"},
+                "NetworkInterfaces" : [{
+                     "AssociatePublicIpAddress" : "True",
+                     "DeleteOnTermination" : "True",
+                     "SubnetId" : {"Ref" : "BootstrapSubnet"},
+                     "DeviceIndex" : "0",
+                     "GroupSet" : [{ "Ref": "BootstrapSecurityGroup" }]
+                }],
+                "Tags": [{ "Key": "Application", "Value": { "Ref": "AWS::StackId" } },
+                         { "Key": "Name", "Value": { "Ref": "AWS::StackName" } }],
+                "UserData": { "Fn::Base64": { "Fn::Join": [ "", [
+                        "#!/bin/bash -xe\n",
+                        "yum install -y git jq\n",
+                        "git clone ",
+                            {"Ref": "DromedaryRepo"},
+                        " /opt/dromedary\n",
+                        "aws sts assume-role --role-arn ",
+                            {"Fn::GetAtt": ["BootstrapInstanceRole", "Arn"]},
+                        " --role-session-name bootstrap > creds.json\n",
+                        "export AWS_DEFAULT_REGION=\"us-east-1\"\n",
+                        "export AWS_DEFAULT_OUTPUT=\"json\"\n",
+                        "export AWS_ACCESS_KEY_ID=`jq -r '.Credentials.AccessKeyId' < creds.json`\n",
+                        "export AWS_SECRET_ACCESS_KEY=`jq -r '.Credentials.SecretAccessKey' < creds.json`\n",
+                        "export AWS_SESSION_TOKEN=`jq -r '.Credentials.SessionToken' < creds.json`\n",
+                        "cd /opt/dromedary\n",
+                        "./bin/bootstrap-all.sh ",
+                          {"Ref": "ProdHostedZone"},
+                        "\n",
+                        "sleep ",
+                          {"Ref": "AliveDuration"},
+                        "\n",
+                        "yes | ./bin/delete-all.sh\n",
+                        "aws cloudformation delete-stack --stack-name ",
+                            { "Ref": "AWS::StackName" },
```

Comment:
You may want to delete the bootstrap stack only if `delete-all.sh` completes successfully.